docs/concepts/pwa-building-blocks.md

```diff
@@ -0,0 +1,105 @@
+Each of these is seperately configurable (see [Building and Running nginx Docker Image](../guides/nginx-startup.md)).
+Nginx enables the following features to be used in a Intershop PWA deployment:
+
+- Uncomplicated caching of PWA server-side rendering responses provided by the upstream Angular Universal server.
+- Integration of the [PageSpeed Module](https://www.modpagespeed.com/) for access to different browser optimizations.
+- Handling of multiple channels via URL parameters in conjunction with SSR (see [Multi-Site Handling](multi-site-handling.md)).
+- Customizable compression for downstream services
+- Device type detection to ensure a correct pre-render, adapted to the incoming user agent.
+
+For an overview of the ever-growing list of third party integrations relating to nginx and deployment in general, see [Third-party Integrations](../README.md#third-party-integrations).
+
+## Browser
+
+The browser runs the bootstrapped/pre-rendered Angular application.
+After initially communicating with the _nginx_ webserver, later REST request are directed to the configured ICM endpoint or custom backend.
+For more information on the browser's role in rendering the Intershop PWA, see [Deployment Scenarios for Angular Applications](deployment-angular.md).
+
+# Default Production Deployment
+
+Chaining the building blocks together results in the depicted system.
+Read on for a step-by-step walkthrough of the initial connection request.
+
+![Current Deployment](pwa-building-blocks-production-deployment.svg)
+
+1. The Browser requests the page by URL from the nginx.
+
+   1. If the cache is enabled and a cached response is found, the response is returned immediately, go to 6
+   2. If no cached response is available, the SSR process is triggered.
+
+2. The node express.js server runs Angular Universal pre-rendering for the requested URL.
+
+3. Angular Universal fills the requested page with content retrieved via the ICM REST API.
+
+4. The response is delivered to nginx, where it is also cached if caching is enabled.
+
+5. The response is delivered to the browser.
+
+6. The initial page response is displayed to the user and the Angular Client Application is booting up in the browser.
+
+7. Once booted up, additional REST Calls are directed straight to the ICM and the PWA acts as a single-page application. No further HTML pages are requested.
+
+## Deployment without nginx
+
+Deployment without using nginx is theoretically possible, even though the many useful features of an nginx deployment are obviously forfeited.
+
+> :warning: Enabling [service workers](progressive-web-app.md#service-worker) is not possible without using nginx. The Intershop PWA will not function as intended if you do.
+
+## Hiding the ICM Backend
+
+For security reasons, it may be desirable to hide the backend adress and prevent direct access to it.
+The Intershop PWA supports this functionality and the default deployment uses this feature.
```

Comment:
I think we can be more explicit here:
- "This can be done by setting the `PROXY_ICM` environment variable on the SSR container to a new URL" + Link to ssr-startup.md
- Link to public demo server? Or insert link to root README.md, where demo server link is provided.